backend_new/test/unit/utilities/pagination-helpers.spec.ts

```diff
@@ -37,4 +38,35 @@ describe('Testing pagination helpers', () => {
       expect(calculateTake(1)).toBe(1);
     });
   });
+  describe('Testing buildPaginationMetaInfo', () => {
+    it('should return 1 page for 1 full page', () => {
+      expect(buildPaginationMetaInfo({ limit: 10, page: 1 }, 10, 10)).toEqual({
+        currentPage: 1,
+        itemCount: 10,
+        itemsPerPage: 10,
+        totalItems: 10,
+        totalPages: 1,
+      });
+    });
+    it('should return 2 page for 1 full page', () => {
+      expect(buildPaginationMetaInfo({ limit: 10, page: 1 }, 20, 10)).toEqual({
+        currentPage: 1,
+        itemCount: 10,
+        itemsPerPage: 10,
+        totalItems: 20,
+        totalPages: 2,
+      });
+    });
```

Comment:
The descriptions for these tests are a little confusing, can you please update and also add more test coverage for different variations of params.

Reply:
updated and added a few more tests